vita3k/gui/src/app_selector.cpp

```diff
@@ -184,13 +184,13 @@ static float scroll_type, current_scroll_pos, max_scroll_pos;
 void draw_app_selector(GuiState &gui, HostState &host) {
     const ImVec2 display_size = ImGui::GetIO().DisplaySize;
     const auto scal = ImVec2(display_size.x / 960.0f, display_size.y / 544.0f);
-    const auto MENUBAR_HEIGHT = 32.f * scal.y;
+    const auto INFORMATION_BAR_HEIGHT = 32.f * scal.y;
     const auto MENUBAR_BG_HEIGHT = !gui.live_area.information_bar ? 22.f : 32.f * scal.y;
```

Comment:
AH my bad lol if yçou can change this to 
```suggestion
    const auto MENUBAR_BG_HEIGHT = !gui.live_area.information_bar ? 22.f : INFORMATION_BAR_HEIGHT ;
```
thx